src/Commands/SyncLayouts.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace Plank\Contentable\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\File;
+use Plank\Contentable\Contracts\Layout;
+use Plank\Contentable\Contracts\Layoutable;
+use Plank\Contentable\Enums\LayoutType;
+use SplFileInfo;
+
+class SyncLayouts extends Command
+{
+    protected $signature = 'contentable:sync';
+
+    protected $description = 'Ensure defined layout files exist as Layout Models.';
+
+    public function handle(): void
+    {
+        foreach ($this->globalKeys() as $key) {
+            $this->ensureGlobalLayoutExists($key);
+        }
+
+        foreach ($this->layoutableKeys() as $key) {
+            $this->ensureModelLayoutExists($key);
+        }
+    }
+
+    /**
+     * Get the Layout keys available to all Layoutables
+     */
+    protected function globalKeys(): array
+    {
+        $layoutModel = static::layoutModel();
+
+        return array_map(function (SplFileInfo $layout) use ($layoutModel): string {
```

Comment:
nit: I'm wondering if defining the return type of an anonymous call back that gets processed immediately is overly verbose here, since we already define a return type for the containing function and the callable passed to `array_map()` is never used anywhere else. Thoughts?

```suggestion
        return array_map(function (SplFileInfo $layout) use ($layoutModel) {
```

Reply:
tbh, this was copilot generated code. I personally like typehinting, but had I typed this out myself, I likely would not have spent the effort.